docs/data/base/components/slider/slider.md

```diff
@@ -1,193 +1,214 @@
+The `min` and `max` props can be used to restrict the value(s) within a range.
 
-Hooks _do not_ support [slot props](#custom-structure), but they do support [customization props](#customization).
+```tsx
+<Slider.Root min={1} max={9}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-:::info
-Hooks give you the most room for customization, but require more work to implement.
-With hooks, you can take full control over how your component is rendered, and define all the custom props and CSS classes you need.
+### Step
 
-You may not need to use hooks unless you find that you're limited by the customization options of their component counterparts—for instance, if your component requires significantly different [structure](#anatomy).
-:::
+The `step` prop snaps the each value to multiples of the given number. In the below example, the input value snaps to increments of `4` starting from the initial `defaultValue`: `3`, `7`, `11`, `15`, and so on.
 
-## Customization
+```tsx
+<Slider.Root step={4} defaultValue={3}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-### Vertical
+You can specify the `largeStep` prop to change the step when the user holds the <kbd>shift</kbd> key, snapping to multiples of 10 by default.
 
-Slider components can be arranged vertically as well as horizontally.
+## Range Sliders
 
-When vertical, you must set `orientation="vertical"` on the Slider so the user can navigate with the up and down arrow keys (rather than the default left-to-right behavior for horizontal sliders).
+To let users set the start and end of a range on a Slider, provide an array of values to the `value` or `defaultValue` prop, and place a `<Slider.Thumb />` for each value in the array:
 
-{{"demo": "VerticalSlider.js"}}
+```tsx
+<Slider.Root defaultValue={[45, 70]}>
+  <Slider.Output />
+  <Slider.Track>
+    <Slider.Thumb />
+    <Slider.Thumb />
+  </Slider.Track>
+</Slider.Root>
+```
 
-### Discrete sliders
+{{"demo": "RangeSlider.js"}}
 
-The most basic Slider is _continuous_, which means it does not have pre-defined (_discrete_) values for the user to select from.
-This is suitable for situations in which an approximate value is good enough for the user, such as brightness or volume.
+### Overlapping values
 
-But if your users need more precise options, you can create a discrete Slider that snaps the thumb to pre-defined stops along the bar.
-Make sure to adjust the `shiftStep` prop (the granularity with which the slider can step when using Page Up/Down or Shift + Arrow Up/Down) to a value divadable with the `step`.
+The `minDifferenceBetweenValues` prop can be used to to set the mininum difference between values in a range slider, so thumbs do not overlap in the same position. In the below example, the thumbs cannot be moved further towards each other as the difference between values would exceed `minDifferenceBetweenValues`:
```

Comment:
> What is the “distance” in the minDistanceBetweenValues prop? Is it a step? If so, we should replace “Distance” with “Step” in the prop name.

@colmtuite What about (literally) `minDifference` instead of `minDistance`?

If it's `minSteps`, you'd need to multiply it by the `step` to get the actual difference between values, which is something you said you wanted to avoid before (coupling 2 props together, like `multipler * step` instead of `largeStep`)